python/ray/_raylet.pyx

```diff
@@ -1498,6 +1540,10 @@ cdef class CoreWorker:
     def force_spill_objects(self, object_refs):
         cdef c_vector[CObjectID] object_ids
         object_ids = ObjectRefsToVector(object_refs)
+        assert not RayConfig.instance().automatic_object_deletion_enabled(), (
+            "Automatic object deletion is not supported for"
+            "force_spill_objects yet. Please set"
+            "automatic_object_deletion_enabled: False in Ray's system config.")
```

Comment:
Can we just log warning instead of raising an error here?

Reply:
Imo, the manual spilling just doesn't work at all when this is not enabled, so I think it makes sense to raise an error instead of logging a warning message.